core/client/src/test/java/alluxio/hadoop/AbstractFileSystemTest.java

```diff
@@ -159,6 +159,33 @@ public void hadoopShouldLoadFileSystemWhenConfiguredTest() throws Exception {
     alluxio.client.file.FileSystem.Factory.get();
     ClientTestUtils.resetClientContext();
   }
+  
+  /**
+  * Tests for ALLUXIO-2036.
+  * Cause by ALLUXIO-2025
```

Comment:
The javadoc should describe what is being tested, not the ticket for writing the test